Aptoide/aptoide-client-v8
# [ASV-458] Store theme persistence in Search

**What does this PR do?**

 This PR aims to keep the store themes in the search result when the user searches in his/her store.

**Database changed?**

   No

**Where should the reviewer start?**

SearchResultFragment.java

**How should this be manually tested?**

Do a search in a store with a theme and check if the search keeps the theme.
Navigate to home and confirm that the theme isn't propagated anymore.

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-458

**Code Review Checklist**
